platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -874,6 +872,13 @@ private ImportResponse configureProject(ImportProject importProject, FolderEntry
                 postImportProjectHandler.onProjectImported(project.getBaseFolder());
             }
         } catch (ConflictException | ForbiddenException | ServerException | NotFoundException e) {
+            if (newProject.getDescription() != null
+                || newProject.getAttributes() != null
+                || newProject.getRunners() != null
+                || newProject.getBuilders() != null
+                || newProject.getMixinTypes() != null) {
+                throw new BadRequestException("Impossible to save configurations for the project without type");
```

Comment:
this code looks strange. how the server exception related to the given message ?